Tools/autotest/pysim/util.py

```diff
@@ -197,7 +199,13 @@ def start_SITL(binary, valgrind=False, gdb=False, wipe=False, synthetic_clock=Tr
         f.write("r\n")
         f.close()
         cmd.extend(['xterm', '-e', 'gdb', '-x', '/tmp/x.gdb', '--args'])
-    
+    if gdbserver:
+        f = open("/tmp/x.gdb", "w")
```

Comment:
We really should do better here.  This is, of course, no worse that what's being done in the gdb case.